client/my-sites/site-settings/form-writing.jsx

```diff
@@ -123,6 +124,23 @@ class SiteSettingsFormWriting extends Component {
 						<PressThis />
 					</div>
 				) }
+				{
+					this.props.isJetpackSite && this.props.jetpackSettingsUISupported && (
+						this.renderSectionHeader( translate( 'Media' ) ) )
+				}
+				{
+					this.props.isJetpackSite && this.props.jetpackSettingsUISupported && (
+						<MediaSettings
+							siteId={ this.props.siteId }
+							onSubmitForm={ this.props.handleSubmitForm }
+							handleToggle={ handleToggle }
+							onChangeField={ onChangeField }
+							eventTracker={ eventTracker }
```

Comment:
I don't think so. If we're to implement more tracking, we should be doing in the change handling methods in `wrapSettingsForm`.